rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -762,8 +763,11 @@ private <T> List<JRightPadded<T>> visitRightPadded(ASTNode[] nodes, @Nullable St
         }
 
         private Expression insideParentheses(ASTNode node, Function<Space, Expression> parenthesizedTree) {
+            int saveCursor = cursor;
+            whitespace();
```

Comment:
grabbing whitespace and resetting the cursor is only to determine the parenthesis level correctly, perhaps it should be moved there